### PR TITLE
Has 101 updated security config to allow admins to access all the resources

### PR DESCRIPTION
**Pull request by a-bhavsar**

### CODE REVIEW

**Review by mjoshi-92 — APPROVED** [review]

**Review by mjoshi-92 — APPROVED** [review]

**Review by Dhruv-1105 — APPROVED** [review]

File: Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/AdminDaoImpl.java [hunk 1]
```diff
@@ -77,6 +77,7 @@ public Admin createAdmin(Admin admin) {
         }
         admin.setUser(user);
         admin.getUser().setRole(Role.ADMIN);
+        System.out.println(admin);
```
Comment (SanketRathod666) [inline Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/AdminDaoImpl.java:80]: remove this line

File: Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/RegisterDaoImpl.java [hunk 2]
```diff
@@ -38,13 +39,15 @@ public AuthenticationResponse registerUser(User user) {
...
-        var jwtToken = jwtService.generateToken(createdUser.getUsername());
+        UserDetailForToken userDetailForToken = new UserDetailForToken(createdUser.getEmail(), createdUser.getRole());
+        var jwtToken = jwtService.generateToken(userDetailForToken);
 
         return AuthenticationResponse.builder().token(jwtToken).build();
     }
 
     @Override
     public AuthenticationResponse registerDoctor(Doctor doctor) {
+        System.out.println(doctor);
```
Comment (SanketRathod666) [inline Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/RegisterDaoImpl.java:50]: remove this line

File: Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java [hunk 3]
```diff
@@ -43,23 +43,25 @@ public User getUser(int id) {
     public User createUser(User user) {
         user.setPassword(passwordEncoder.encode(user.getPassword()));
         List<Symptom> symptomList = user.getSymptoms();
+        System.out.println(symptomList);
```
Comment (SanketRathod666) [inline Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java:46]: remove

File: Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java [hunk 4]
```diff
@@ -43,23 +43,25 @@ public User getUser(int id) {
...
-            newSymptomList.add(symptomRepository.findById(symptom.getId()).get());
-        }
-        user.setSymptoms(newSymptomList);
+            for(Symptom symptom : symptomList) {
+                newSymptomList.add(symptomRepository.findById(symptom.getId()).get());
+            }
+            user.setSymptoms(newSymptomList);
         }
         List<PatientChronicIllness> patientChronicIllnessList = user.getPatientChronicIllness();
+        System.out.println(patientChronicIllnessList);
```
Comment (SanketRathod666) [inline Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java:55]: remove

File: Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java [hunk 5]
```diff
@@ -74,6 +76,7 @@ public User updateUser(int id, User user) {
             for(PatientChronicIllness patientChronicIllness : oldUser.get().getPatientChronicIllness()){
                 patientChronicIllnessDao.deletePatientChronicIllness(patientChronicIllness.getId());
             }
+            System.out.println("Hey");
```
Comment (SanketRathod666) [inline Backend/has-docTime/src/main/java/com/spring/hasdocTime/dao/UserDaoImpl.java:79]: remove

**Review by SanketRathod666 — APPROVED** [review]

**Pull request merged**